models/articles.model.js

```diff
@@ -31,4 +31,13 @@ const fetchCommentsByArticleId = (article_id) => {
     })
 }
 
-module.exports = { fetchArticles, fetchArticleById, fetchCommentsByArticleId }
\ No newline at end of file
+const updateArticleVoteCount = (article_id, inc_votes) => {
+    if (typeof inc_votes === 'number'){
+        return db.query(`UPDATE articles SET votes = votes + $1 WHERE article_id = $2 RETURNING *;`, [inc_votes, article_id])
+        .then((result) => {
+            return result.rows[0];
+        })
+    }else return Promise.reject({ status: 400, msg: 'Bad Request: inc_votes must be a number' });
```

Comment:
You may have been able to rely an PSQL to check the value is a number. Also it is common to see the edgecase handled first and then because you are returning you don't need the else.

```js
func f(){
    if(edgeCase){
        return error
    }
    return result
}